data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,604 @@ mission "Heliarchs Buy Jump Drive Mission"
+					goto publishingcompany
+				`	"Perhaps the Saryd. People with human parts are likely to be more interested in human books."`
+					goto saryd
+					to display
+						not "struggling author: saryd"
+				`	"The Arachi have a close-knit population. Word of a new book might spread quickly there."`
+					to display
+						not "struggling author: arachi"
+			action
+				set "struggling author: arachi"
+			`	"Oh, it's kind of embarrassing but... I'm actually a bit afraid of spiders, so I'm going to take a while to get used to the Arachi."`
+				goto questions
+			label saryd
+			action
+				set "struggling author: saryd"
+			`	"I thought the same at first, but from what I've seen the Saryds seem pretty reserved, don't you think? Maybe another species would be better."`
+				goto questions
+			label publishingcompany
+			action
+				clear "struggling author: arachi"
+				clear "struggling author: saryd"
+			`	"But where can we find a Kimek publishing company? We haven't even translated our book into their language yet!"`
+			choice
+				`	"We'll find a company."`
+				`	"Given the number of books here, there has to be a publishing company somewhere."`
+			`	"The faster we can get the book published, the better. Meet me in the spaceport in a few hours; I'll find a good company."`
+			`	Before Anna can head off, the Heliarch rises. "Again accompany you, I will."`
+
+
+mission "Coalition: Struggling Author 4"
+	name "Kimek Publishing Company"
+	description "Anna found a Kimek publishing company who is willing to take in Anna Fangvin to get her book published. Bring Anna Fangvin to <destination>."
+	source "Ki Patek Ka"
+	destination "Inmost Blue"
+	blocked "You do not have enough bunk space for Anna Fangvin and the Heliarch agent. Come back here when you have two bunks free."
+	passengers 2
+	to offer
+		has "Coalition: Struggling Author 3: done"
+	on offer
+		conversation
+			`The spaceport of Ki Patek Ka is vast, and would normally prove a nightmarish place to look for just one person. Thankfully, Anna being one of the few humans within the Coalition makes your search relatively quick. She has chosen to settle in a cafeteria with her laptop. She is typing away at the keyboard, presumably writing a short story to post.`
+			`	"<first>! I found a company who's willing to publish me! I sent them a digital copy of my book and they instructed me to visit their office on <planet>," explains Anna, once again enthusiastic.`
+			choice
+				`	"Great!"`
+					goto go
+				`	"Let's go!"`
+					goto go
+				`	"What's the publishing company called?"`
+			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a blue dolphin.`
+			label go
+			`	Anna heads to your ship, and you plot a course to <destination>. Just before you launch, the Heliarch hops in.`
```

Comment:
The Heliarch just hopping in last minute is odd.

I think we could:

- Have it so that he's again helping Anna when you find her, and follows you two to the ship, or
- Have it so that he had gone to look for info elsewhere, and when you and Anna are returning to your ship  you find him waiting there, and catch up briefly just before boarding and launching.